types/webrtc/RTCPeerConnection.d.ts

```diff
@@ -339,7 +339,7 @@ interface RTCPeerConnection extends EventTarget {
         failureCallback: RTCPeerConnectionErrorCallback): Promise<void>;
 }
 interface RTCPeerConnectionStatic {
-    new(configuration?: RTCConfiguration): RTCPeerConnection;
+    new(configuration?: RTCConfiguration, options?: any): RTCPeerConnection;
```

Comment:
Hm, there's no `options` parameter in the associated [IDL](https://www.w3.org/TR/webrtc/#idl-def-rtcpeerconnection); do you have a source?

Reply:
@weswigham I understand the RTCPeerConnection spec doesn't have this defined. However, Chrome still supports passing the additional optional constraints, for e.g. to disable the default DTLS and enable SDES encryption. Having this optional parameter provides flexibility to developers who may like to test the feature by switching between different encryption mechanisms.

Reply:
There a link to the chrome docs for it?